plugins/language_conf.lua

```diff
@@ -0,0 +1,46 @@
+-- mod-version:3
+
+local syntax = require "core.syntax"
+
+syntax.add {
+  name = "Conf",
+  files = { ".gitconfig", ".rclone.conf", "%.conf$" },
+  comment = '#',
+  patterns = {
+    { pattern = ";.-\n", type = "comment" },
+    { pattern = "#.-\n", type = "comment" },
+    { pattern = { "%[", "%]" }, type = "keyword" },
+
+    { pattern = { '"""', '"""', '\\' }, type = "string" },
+    { pattern = { '"', '"', '\\' }, type = "string" },
+    { pattern = { "'''", "'''" }, type = "string" },
+    { pattern = { "'", "'" }, type = "string" },
+    { pattern = "[A-Za-z0-9_%.%-]+%s*%f[=]", type = "function" },
+    { pattern = "%s+%-%-[A-Za-z0-9%-]+", type = "normal" },
+    -- Git
+    { pattern = "HEAD", type = "literal" },
+    -- Rclone AWS S3 canned ACLs
```

Comment:
I'm wondering if there should be multiple syntaxes at this point.

Reply:
Thanks for the review.  
They could be separate syntax files, but at the same time all these conf files are very alike – just INI files with slightly different keywords.